subsys/bluetooth/controller/ll_sw/ull_sync.c

```diff
@@ -349,8 +350,24 @@ uint8_t ll_sync_terminate(uint16_t handle)
 
 uint8_t ll_sync_recv_enable(uint16_t handle, uint8_t enable)
 {
-	/* TODO: Add support for reporting enable/disable */
-	return BT_HCI_ERR_CMD_DISALLOWED;
+	struct ll_sync_set *sync;
+
+	sync = ull_sync_is_enabled_get(handle);
+	if (!sync) {
+		return BT_HCI_ERR_UNKNOWN_ADV_IDENTIFIER;
+	}
+
+	/* Reporting enabled/disabled */
+	sync->rx_enable = (enable & BT_HCI_LE_SET_PER_ADV_RECV_ENABLE_ENABLE) ?
+			  1U : 0U;
+
+#if defined(CONFIG_BT_CTLR_SYNC_PERIODIC_ADI_SUPPORT)
+	sync->nodups = (enable &
+			BT_HCI_LE_SET_PER_ADV_RECV_ENABLE_FILTER_DUPLICATE) ?
+		       1U : 0U;
```

Comment:
It's OK to go past 80 characters (and below 100) if it improves readability significantly, which I think this does:

```suggestion
	sync->nodups = (enable & BT_HCI_LE_SET_PER_ADV_RECV_ENABLE_FILTER_DUPLICATE) ?
		       1U : 0U;
```

Reply:
Fixed